solutions/cfgym-102951C.mdx

```diff
@@ -74,4 +76,57 @@ int main() {
 
 </CPPSection>
 
+<JavaSection>
+	
+``` java
+import java.io.*;
+
+import java.util.*;
+
+
+public class Main {
+	// CodeSnip{Kattio}
+	public static void main(String[] args) {
+		Kattio io = new Kattio();
+
+		int n = io.nextInt();		
+		int[] a = new int[n + 1], b = new int[n + 1], c = new int[n + 1], lis = new int[n + 1];
```

Comment:
Maybe more descriptive names?

Reply:
Hi, I've added in descriptive names wherever possible and I think we should be leaving the names of A and B arrays as it is as that's how it was given in the problem description over [here](https://codeforces.com/gym/102951/problem/C).
Lemme know if I should make any more changes.